docker-compose.yml

```diff
@@ -72,6 +72,21 @@ services:
       - '1977:1977'
     networks:
       - default
+  pattern-atlas-ui:
+    image: patternpedia/patternrepo-ui:latest
```

Comment:
should be patternatlas/...
shouldn't it @manuwei?

Reply:
I got error when I changed patterpedia to patternatlas.
ERROR: pull access denied for patternatlas/patternrepo-ui, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

Reply:
the [PR](https://github.com/PatternAtlas/pattern-atlas-docker/pull/5/files) in PatternAtlas is now merged and you can try it again :)

Reply:
done